app/routes.php

```diff
@@ -130,4 +130,5 @@
     $app->post('/api/addTrainer', AddTrainerController::class);
     $app->delete('/api/deleteTrainer', DeleteTrainerController::class);
     $app->post('/api/sendEmail', SendEmailController::class);
+    $app->get('/api/lockApplicantField',\Portal\Controllers\API\LockApplicantFieldController::class);
```

Comment:
can you move the namespace into a use statement so this route looks consistent with the others